relay-server/src/actors/outcome.rs

```diff
@@ -28,15 +28,19 @@ use crate::ServerError;
 // Processing outcome implementation
 #[cfg(feature = "processing")]
 pub use self::processing::*;
-// non-processing outcome implementation
+
+#[cfg(feature = "processing")]
+pub type OutcomeProducer = processing::ProcessingOutcomeProducer;
 #[cfg(not(feature = "processing"))]
-pub use self::non_processing::*;
+pub type OutcomeProducer = HttpOutcomeProducer;
+
 use std::borrow::Cow;
+use std::mem;
```

Comment:
Please move these imports up to the other `std` imports to match the usual import order.

Reply:
yep